### PR TITLE
Update system_setup.rst

### PR DESCRIPTION
**Pull request by nielsdrost**
Added a note that a codified version of these instructions is available in the infra repo

### CODE REVIEW

File: docs/system_setup.rst [hunk 1]
```diff
@@ -6,6 +6,8 @@ and data.
 
 This chapter is for system administrators or Research Software Engineers who need to set up a system for the eWaterCycle platform.
 
+These instructions cover installing an eWaterCycle system from scratch on an "empty" Linux machine. We have also have created a codified version of these instructions using `Ansible <https://docs.ansible.com/ansible/latest/index.html>`__ specifically targeted at the `SURF Research Cloud <https://servicedesk.surfsara.nl/wiki/display/WIKI/Research+Cloud+Documentation>`__ in a `separate Infra repo <https://github.com/eWaterCycle/infra>`__.
+
```
Comment (RolfHut) [inline docs/system_setup.rst:10]: Maybe add a (very) short list of requirements for setting up a machine on the research cloud, like: "starting a machine on the Surf Research Cloud requires that you have research budget with SURF, for more info see the website of `SURF<https://www.surf.nl/en/research-it/apply-for-access-to-compute-services>`

**Review by RolfHut — DISMISSED** [review]
see comment (I need to find out how to directly change this myself...)

File: docs/system_setup.rst (hunk 1 above)
Comment (nielsdrost) [reply docs/system_setup.rst:10]: Added the note to the Infra repo readme as I think it makes more sense there.

**Review by Peter9192 — APPROVED** [review]
LGTM

**Review by RolfHut — APPROVED** [review]

**Pull request merged**